extensions/wikia/UserLogin/js/spec/UserBaseAjaxForm.spec.js

```diff
@@ -0,0 +1,54 @@
+/* global UserBaseAjaxForm */
+
+describe('Base class for User Ajax Forms', function () {
+	'use strict';
+
+	var instance, $el, options,
+		proto = UserBaseAjaxForm.prototype,
+		$userNameInput = $('<input type="text" name="username">'),
+		$form = $('<form></form>').append($userNameInput);
+
+	window.WikiaForm = function () {
+		return {
+			form: $form,
+			inputs: {
+				username: $userNameInput
+			}
+		};
+	};
+
+	beforeEach(function () {
+		$el = $('<div></div>').append($form);
+		options = {
+			skipFocus: true
+		};
+	});
+
+	it ('Has base methods that other classes depend on', function () {
```

Comment:
Should there be a space after `it`? It's not an anonymous function, and I've seen it without the space elsewhere so I'm guessing not?